src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -211,8 +211,51 @@ module sonic-flex_counter {
                 }
             }
 
+            container FLOW_CNT_ROUTE {
+                /* ROUTE_FLEX_COUNTER_GROUP */
+                leaf FLEX_COUNTER_STATUS {
+                    type flex_status;
+                }
+                leaf FLEX_COUNTER_DELAY_STATUS {
+                    type flex_delay_status;
+                }
+                leaf POLL_INTERVAL {
+                    type poll_interval;
+                }
+            }
+
         }
         /* end of container FLEX_COUNTER_TABLE */
+
+        container FLOW_COUNTER_ROUTE_PATTERN_TABLE {
+            description "Flow counter route pattern of config_db.json";
+
+            list PATTERN_LIST {
+
+                key "vrf_name ip_prefix";
+
+                leaf vrf_name {
+                    type string {
+                        length 0..16;
+                    }
+                }
+
+                leaf ip_prefix {
+                    type union {
```

Comment:
Can we use the sonic-ip-prefix type defined here?
https://github.com/Azure/sonic-buildimage/blob/e88d7b3b934cea4a279fd99965c142076ea83c18/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L47